odoo/sql_db.py

```diff
@@ -744,6 +745,7 @@ def borrow(self, connection_info):
         result._original_dsn = connection_info
         self._connections.append((result, True))
         self._debug('Create new connection')
+        _logger.info("Connection backend PID %d", result.get_backend_pid())
```

Comment:
Since there's already a log message, the new information should be added there.

Reply:
Ok, I add the additional information and I changed to INFO level in order to get this info for all cases

Reply:
Can you explain the use case?

Maybe we could move this to a sub-logger instead so the current behaviour doesn't change (in environments with lots of dbs or a high worker cycling rate I'd expect it to show up a lot) but it's easy to configure separately without having all the spam of logging every SQL query? (which I can understand makes turning on `odoo.sql_db:DEBUG` difficult in production)

Reply:
Done!

`odoo-bin ... --log-handler=odoo.sql_db.connection:DEBUG`

`2022-01-19 16:24:58,096 12877 DEBUG ? odoo.sql_db.connection: Create new connection backend PID 12881`

Reply:
The use case is when postgresql.log shows errors like:

 - [394452]: ... LOG:  XX00: Could not receive data from client: Connection time out

So, you can looking for what was the Odoo PID related with PostgreSQL connection 394452

In my case it was the Odoo-PID of longpolling so the chat was stuck but without extra info it was so hard to know where is the problem since that postgresql show you the PostgreSQL-PID but Odoo even don't show who is using it

At final we realized that GCP needs an extra configuration in TCP-keepalive to allow connection opened a long time (as longpolling requires)

The same was using `queue` module but then they already have fixed:
 - https://github.com/OCA/queue/pull/249
   - https://github.com/OCA/queue/issues/246

(IMHO longpolling requires a method similar to keep-alive but it is another history)

This PR will help to debug the issues better